elliott/scripts/jira_issue_tool.py

```
@@ -0,0 +1,331 @@
+        'CVE ID': None,
+        'Downstream Component Name': None,
+        'Embargo Status': None,
+        'Release Notes Text': None,
+        'Release Notes Type': None,
+    }
+
+    # Resolve field IDs
+    id_to_name = field_map
+    name_to_id = {v: k for k, v in id_to_name.items()}
+
+    print(f"\n--- ART-relevant Custom Fields ---")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)